docs/src/1_Mechanics/04_Powers.yaml

```
@@ -609,13 +609,118 @@ Handy:
             - Craft > 1
             - Knowledge > 1
 
+Small Power Gain:
+    Type: Passive
+    Category:
+        - Power Points
+    Mechanic: You gain 1 additional Power Point.
+    StatAdjust:
+        Add:
+            PP: 1
+    Description: Harnessing more stored power allows for greater achievements.
+    XP: 1
+
+Large Power Gain:
+    Type: Passive
+    Category:
+        - Power Points
+    Mechanic: You gain 3 additional Power Points.
+    StatAdjust:
+        Add:
+            PP: 3
+    Description: Gathering power is one of the paths to true mastery of the self.
+    XP: 2
+
 Craft Light:
+    Type: Minor
     Category:
         - Magic
     Description: Your skills let you conjure illusory radiance.
-    Mechanic: You can create or infuse a temporary illusory object with light in an Aura 6 radiance. If the light is more than 100 spaces away from you, it disappears. You may end this effect as a free action.
+    Mechanic: You can create or infuse a temporary illusory object with light in an Aura 6 radiance. If the light is more than 100 spaces away from you, it disappears. You may end this effect as a free action. Additionally, you may expend 1 PP to force a target within 6 spaces to make a DR 2 GUT Save or become Blinded for 1 round.
+    XP: 1
+    PP: [0, 1]
+    Save:
+        Trigger: Once
+        DR: 2
+        Type: GUT
+        Fail: Blinded
```

By including the save in both the mechanic description and as a field, the markdown would render it twice. Under the current system, powers should describe saves in only one place. This is an issue with other new powers below.

I also describe the effects of PP use independently. The use of 'additionally' was ambiguous to me as to whether or not these things happened sequentially

```suggestion
    Mechanic: For 0 PP, you infuse an inanimate object with light, spanning Aura 6. If the light is more than 10 spaces away from you, it disappears. You may end this effect as a free action. For 1 PP, you may target one creature with this effect. When targeted, a creature makes a DR 2 GUT Save. On a failure, they are Blinded for 1 round.
    XP: 1
    PP: [0, 1]
```